vignettes/baggr.Rmd

```diff
@@ -294,24 +296,22 @@ forest_plot(baggr_schools, show = "both")
+_Baggr_ has built-in, automated leave-one-out cross validation for its models. The values returned by `loocv()` can be used to understand how any
+one group affects the overall result, as well as how well the model predicts the omitted group.
 
-```{r loocv, eval = FALSE}
-loocv_res <- loocv(schools, return_models = FALSE, "rubin", pooling = "partial")
-```
+This function automatically runs $K$ baggr models, leaving out one group at a time, and then calculates expected log predictive density (ELPD) for that group (see Gelman et al 2013). The main output is the cross-validation information criterion, or -2 times the ELPD averaged over $K$ models. This is related to, and often approximated by, the Watanabe-Akaike Information Criterion. A value closer to zero (i.e. a smaller number in magnitude) means a better fit. For more information on cross-validation see this [overview article](http://www.stat.columbia.edu/~gelman/research/published/waic_understand3.pdf).
+
+Therefore, if you have $K$ groups, using the `loocv()` will run your model of choice $K$ times. Be aware that this may take a while even for simple models. (You should see a progress bar in your terminal window.) The `loocv()` function takes in the same arguments as `baggr()`, plus an option (`return_models`) for whether to return all the models or just the summary statistics. For the 8 schools example we can do
 
-```{r echo = FALSE, include = FALSE, eval = TRUE}
-#this is ugly but effective way to stop output from loocv getting printed
+```{r loocv, echo = T, results = 'hide', warning = F, message = F}
 loocv_res <- loocv(schools, return_models = FALSE, "rubin", pooling = "partial")
 ```
 
 ```{r}
 loocv_res
 ```
 
-The main output is $-2$ times the log predictive density (_lpd_) summed over 8 models, which corresponds to the Watanabe-Akaike Information Criterion. A WAIC value closer to zero (i.e. a smaller number in magnitude) means a better fit. 
-
-More data are stored in `loocv()` output, and can be accessed via `attributes()`, e.g. the mean treatment effects, their variability and _lpd_ for each model that are stored in the attribute `df`:
+The main output is $-2$ times the log predictive density (_ELPD_) summed over 8 models, which corresponds to the Watanabe-Akaike Information Criterion. The `loocv()` output contains more than just the matrix it prints, and this additional information can be accessed via `attributes()`, e.g. the mean treatment effects, their variability and _elpd_ for each model that are stored in the attribute `df`:
```

Comment:
Couple lines above it ELPD is defined and abbreviated, and here we say "log predictive density (_ELPD_)"... And the -2 thing repeats info from the paragraph directly above.

Reply:
This makes total sense, I'll fix this.